builtin/providers/aws/resource_aws_redshift_cluster_test.go

```diff
@@ -339,6 +372,48 @@ resource "aws_redshift_cluster" "default" {
   allow_version_upgrade = false
 }`
 
+var testAccAWSRedshiftClusterConfig_tags = `
+provider "aws" {
+	region = "us-west-2"
+}
```

Comment:
I think the `provider` block doesn't need to be here? At least I don't remember seeing it anywhere in other acceptance tests, so I assume it's handled globally for all AWS acceptance tests.